### DISCUSSION TITLE
TIC80 using many sound outputs on my Focusrite audio interface

### DISCUSSION THREAD

**Discussion by jeenio** [nesbox/TIC-80#2138] [Q&A] [answered]
I've been having this issue with 1.0 (I don't recall having it with 0.90) that TIC-80 decides to route audio to many sound outputs in my Focusrite Scarlett 18i20 1st gen. I tested with other apps that use SDL (like schismtracker) and they behave normally, ie. using the default stereo pair available in Windows.

The VERY weird thing is that it sends to the default stereo pair (outputs 1-2), then output 3 (no 4???) and outputs 5-6 and 7-8. I like to have many of these "DAW" outputs available as I have different things coming in from different places, so it bothers me a bit having to mute all these outputs.

Win10 Home 22H2
TIC-80 1.0-2164 Pro
Focusrite Scarlett 18i20 1st gen

**Comment by nesbox** [comment, marked as answer]
Could you pls test the latest dev build, you can download it here https://nightly.link/nesbox/TIC-80/workflows/build/main

**Comment by jeenio** [comment]
Yeah, it's fixed! Awesome! Thanks :)
